src/assets/scripts/client/navigationLibrary/StandardRoute/StandardRouteModel.js

```diff
@@ -291,6 +291,24 @@ export default class StandardRouteModel extends BaseModel {
     }
 
     /**
+     * Returns an array of fix names used by any portion of the procedure
+     *
+     * @for StandardRouteModel
+     * @method getAllFixNames
+     * @return {array<string>} ['fixname', 'fixname', 'fixname', ...]
+     */
+    getAllFixNames() {
```

Comment:
expanded into a new array with spread operators to combine everything, then uniq-ed it at the end.

done